services/w3c/w3c-validation-helper.js

```diff
@@ -77,72 +77,20 @@ const getSchema = preset => {
   return schema.map(url => encodeURI(url)).join(' ')
 }
 
-const documentation = `
```

Comment:
It looks like I'm deleting loads of docs here, but I've just moved most of this into parameter descriptions